src/components/FrozenFunds/Transfer.jsx

```diff
@@ -51,6 +52,7 @@ class Transfer extends React.Component {
 		return (
 			<TransactionScenario
 				handleTransaction={() => this.props.freezeBalance()}
+				onUnlock={() => this.props.clearForm()}
```

Comment:
After user enter password they still can cancel transaction and in this case fields must be fulfilled not be empty